### ISSUE TITLE
PIL.UnidentifiedImageError: cannot identify image file <_io.BytesIO object at 0x7ff81409e930> when using controlnet --control-mode depth

### ISSUE THREAD

**Issue by bluusun** [brycedrennan/imaginAIry#373]
```
Traceback (most recent call last):
  File "/usr/local/bin/imagine", line 7, in <module>
    imaginairy.cli.imagine.imagine_cmd()
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/imaginairy/cli/imagine.py", line 137, in imagine_cmd
    ControlNetInput(
  File "pydantic/main.py", line 339, in pydantic.main.BaseModel.__init__
  File "pydantic/main.py", line 1076, in pydantic.main.validate_model
  File "pydantic/fields.py", line 884, in pydantic.fields.ModelField.validate
  File "pydantic/fields.py", line 1101, in pydantic.fields.ModelField._validate_singleton
  File "pydantic/fields.py", line 1157, in pydantic.fields.ModelField._apply_validators
  File "pydantic/class_validators.py", line 337, in pydantic.class_validators._generic_validator_basic.lambda13
  File "/usr/local/lib/python3.10/site-packages/imaginairy/schema.py", line 105, in validate
    return cls(img=cls.load_image_from_base64(v))
  File "/usr/local/lib/python3.10/site-packages/imaginairy/schema.py", line 122, in load_image_from_base64
    return Image.open(io.BytesIO(img_bytes))
  File "/usr/local/lib/python3.10/site-packages/PIL/Image.py", line 3283, in open
    raise UnidentifiedImageError(msg)
PIL.UnidentifiedImageError: cannot identify image file <_io.BytesIO object at 0x7ff81409e930>
```
This one gives an error:

```
imagine  "$mprompt, stunning colors, zoom out, hyperrealism, perfect light and details,highly detailed, photorealistic, trending on instagram,stunning, 8K,sharp focus, high-resolution"  --control-image 1.jpg  --init-image 1.jpg --control-mode depth --init-image-strength "0.3" -w 1200 -h 768 --model SD-1.5  -r 5 --steps 80 --upscale 
```

This one works:
```
 imagine  "$mprompt, stunning colors, zoom out, hyperrealism, perfect light and details,highly detailed, photorealistic, trending on instagram,stunning, 8K,sharp focus, high-resolution"   --control-image 1.jpg  --init-image 1.jpg  --init-image-strength "0.3" -w 1200 -h 768 --model SD-1.5  -r 5 --steps 80 --upscale 
```

![jason-briscoe-GliaHAJ3_5A-unsplash](https://github.com/brycedrennan/imaginAIry/assets/65051271/9358a6c5-4c49-486b-97ca-d777264b604e)

**Comment by bluusun** [comment]
Seems to happen only in latest version available with pip3 install imaginairy -U

**Comment by brycedrennan** [comment]
hmmm. i'll take a look

**Comment by brycedrennan** [comment]
Okay try just released version `13.2.0`. I'll reopen if its still not fixed.
